testbed/tests/trace_test.go

```diff
@@ -510,3 +505,89 @@ func TestTraceAttributesProcessor(t *testing.T) {
 		})
 	}
 }
+
+func TestTraceAttributesProcessorJaegerGRPC(t *testing.T) {
```

Comment:
Do we really need it given that we test a processor here and Jaeger protocol is going to be replaced with OTLP? Maybe just remove it?

Just to consider. Otherwise LGTM